ui/src/templates/components/CommunityTemplateImportOverlay.tsx

```diff
@@ -26,6 +28,7 @@ interface DispatchProps {
 }
 
 interface StateProps {
+  flags: FlagMap
```

Comment:
i'm confused by your statements this morning. using connect to trigger a rerender on your component on update was what was broken with feature flags?

Reply:
>using connect to trigger a rerender on your component on update was what was broken with feature flags?

@drdelambre i don't think so. there's either a race condition between the `GetFlags` component and the code running behind the `isFlagEnabled` call, or there's something deeper going on with rendering subchildren inside of `Route` components. 

Essentially for flags that require api access, calling `isFlagEnabled` at the route layer in `index.tsx` hasn't worked for me for any flag. you can verify this on latest master, toggle community flags on by running influxd with the flag enabled:

```sh
go run ./cmd/influxd --assets-path=ui/build --feature-flags=communityTemplates=true```

make sure your local storage is clear just in case. see if you can get the flagged version working. if it is working, flip the flag to `false` when running influxd and see if you can get it toggling off.